src/main/java/com/google/api/tools/framework/aspects/versioning/model/VersionAttribute.java

```diff
@@ -16,14 +16,14 @@
 
 package com.google.api.tools.framework.aspects.versioning.model;
 
+import com.google.api.tools.framework.model.Interface;
```

Comment:
Javadoc fails without the import because Interface is not in the same Java package.